### PR TITLE
upgrade to material ui v4

### LINKED ISSUE #219: Move to MaterialUi 4

### PR DESCRIPTION
**Pull request by atcampbell**
fixes #219

### CODE REVIEW

File: src/Service.Host/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "collectCoverageFrom": [
             "client/src/**/*.{js}"
         ],
-        "collectCoverage": true,
+        "collectCoverage": false,
```
Comment (richardiphillips) [inline src/Service.Host/package.json]: did you mean to switch coverage off?

File: src/Service.Host/package.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
     "author": "",
     "license": "ISC",
     "devDependencies": {
+        "autoprefixer": "6.3.7",
```
Comment (richardiphillips) [inline src/Service.Host/package.json]: should I bring all these package.json changes into the template?

Comment (atcampbell) [reply src/Service.Host/package.json]: Yeah, this one was breaking the builds after updating.

File: src/Service.Host/package.json (hunk 1 above)
Comment (atcampbell) [reply src/Service.Host/package.json]: Was doing that in testing, forgot to turn it back on...

**Pull request merged**